.github/workflows/changelogs/test_changelogs.py

```diff
@@ -0,0 +1,383 @@
+    assert ("tckr#01", "01") in entry.trackers["tckr"]
+    assert ("tckr#02", "02") in entry.trackers["tckr"]
+
+def test_validate_chlog_file_valid(validator, chlog_file):
+    chlog_file.write_text("- This is a valid\n  multiline changelog entry\n")
+    issues, entries = validator.validate_chlog_file(str(chlog_file))
+    assert not issues, issues_to_str(issues, 0)
+    assert len(entries) == 1
+
+def test_validate_chlog_file_multiple_entries(validator, chlog_file):
+    chlog_file.write_text("- This is a valid\n  multiline changelog entry\n- This is a second entry\n")
+    issues, entries = validator.validate_chlog_file(str(chlog_file))
+    assert not issues, issues_to_str(issues, 0)
+    assert len(entries) == 2
+
+def test_validate_chlog_file_multiple_entries_line_numbers(validator, chlog_file):
+    chlog_file.write_text("- This is a valid\n  multiline changelog entry\n- This is a second entry\n- a multiline\n  entry\n")
+    issues, entries = validator.validate_chlog_file(str(chlog_file))
+    assert len(issues) == 1, issues_to_str(issues, 1)
+    assert len(entries) == 3
+    issue = issues[0]
+    assert IssueType.WRONG_CAP in str(issue)
+    assert issue.line == 4
+    assert issue.end_line == 5
+
+def test_validate_chlog_file_empty_file(validator, chlog_file):
+    issues, entries = validator.validate_chlog_file(str(chlog_file))
+    assert len(issues) == 1
+    assert IssueType.EMPTY_CHLOG in str(issues[0])
+
+def test_validate_chlog_file_multiple_issues_and_entries(validator, chlog_file):
+    content = """- This is a valid entry
+- This is a valid
+  multiline entry
+- This entry has an extra
+  whitespace  at the second line
+- Invalid entry with multiple trailing whitespaces at the end  
+- Invalid entry: wrong capitalization after the colon
+"""
+    chlog_file.write_text(content)
+    issues, entries = validator.validate_chlog_file(str(chlog_file))
+    assert len(issues) == 3, issues_to_str(issues, 3)
+    assert len(entries) == 5
+
+# Tests for the basic rules
+
+@pytest.mark.parametrize("entry_text, issue_msg", [
+    ("- This entry has\n  trailing whitespaces \n", IssueType.TRAIL_WHITESPACE),
+    ("- This entry has an  extra whitespace\n", IssueType.MULTI_WHITESPACE),
+    (" - This is an invalid changelog entry\n", IssueType.WRONG_INDENT),
+    ("- This is an invalid changelog entry\n This line has only 1 leading whitespace instead of 2\n", IssueType.WRONG_INDENT),
+    ("This changelog entry doesn't start with '- ' characters\n", IssueType.WRONG_START),
+    ("- This is an invalid changelog entry without a newline at the end", IssueType.MISSING_NEWLINE),
+    ("- This entry\n\n  has an empty line in between\n", IssueType.EMPTY_LINE),
+    ("- this entry has wrong capitalization\n", IssueType.WRONG_CAP),
+    ("- This entry has wrong capitalization\n  in the. second sentence\n", IssueType.WRONG_CAP),
+    ("- This entry has wrong capitalization: right here.\n", IssueType.WRONG_CAP),
```

Comment:
According to https://www.grammarly.com/blog/capitalization-after-colons/, the first word after a colon is only capitalized under certain circumstances.

Reply:
Hmm.. let me just remove this then :D